go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/pkg/sftp v1.12.0 // indirect
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
-	github.com/zclconf/go-cty v1.5.1
+	github.com/zclconf/go-cty v1.5.1 // indirect
```

Comment:
It looks like we need to run `go mod tidy` since the modules were in a bit of an inconsistent state when I looked at them locally. When I ran it, it cleaned up some things.